src/github.com/getlantern/detour/whitelist.go

```diff
@@ -15,47 +14,56 @@ var (
 	whitelist   = make(map[string]wlEntry)
 )
 
-func InitWhitelist(wl map[string]time.Time) {
```

Comment:
After changed `AddToWl` and `RemoveFromWl` to public, `InitWhitelist` no longer required.